Makefile

```diff
@@ -42,7 +42,7 @@ chart_setup:
 	mkdir -p ${CHART_PATH}/.packaged
 
 chart_package:
-	echo "appVersion: ${IMAGE_VERSION}" >> ${CHART_PATH}/${CHART_NAME}/Chart.yaml
+    echo "appVersion: ${IMAGE_VERSION}" >> ${CHART_PATH}/${CHART_NAME}/Chart.yaml
```

Comment:
The indenting is off on this line. Also, there's already an appVersion in Chart.yaml so this just adds another appVersion. The appVersion needs to be removed from Chart.yaml.

Reply:
Good catch. Fixed.